arch/arm/src/stm32/hardware/stm32_fdcan.h

```diff
@@ -43,7 +43,6 @@
 
 #define STM32_FDCAN_CREL_OFFSET    0x0000  /* FDCAN core release register */
 #define STM32_FDCAN_ENDN_OFFSET    0x0004  /* FDCAN endian register */
-                                           /* 0x0008 Reserved */
```

Comment:
why removed ?

Reply:
I misread a reviewer's change request above (the change was for the _other_ `stm32_fdcan.h`).  Will revert.